lib/modules/platform/github/index.ts

```diff
@@ -1171,23 +1170,22 @@ export async function setBranchStatus({
 
 /* istanbul ignore next */
 async function getIssues(): Promise<Issue[]> {
-  const result = await githubApi.queryRepoField<Issue>(
-    getIssuesQuery,
-    'issues',
-    {
-      variables: {
-        owner: config.repositoryOwner,
-        name: config.repositoryName,
-        user: config.renovateUsername,
-      },
-    },
-  );
-
-  logger.debug(`Retrieved ${result.length} issues`);
-  return result.map((issue) => ({
-    ...issue,
+  const result: Issue[] = (
+    await githubApi.getJson<Issue[]>(
+      `repos/${config.parentRepo ?? config.repository}/issues?creator=${
+        config.renovateUsername
+      }&state=all`,
+      { repoCache: true },
+    )
+  ).body.map((issue: any) => ({
```

Comment:
```suggestion
  ).body.map((issue) => ({
```
don't use any here, that will kill all type safety